packages/lab/src/pagination/GoToInput.tsx

```diff
@@ -71,28 +77,18 @@ export const GoToInput = forwardRef<HTMLSpanElement, GoToInputProps>(
       setInputValue("");
     }, [setInputValue]);
 
-    useEffect(() => {
-      if (compact) {
-        setInputValue("");
-      }
-    }, [compact]);
-
     const [position, setPosition] = useState<"left" | "right">();
 
     useIsomorphicLayoutEffect(() => {
-      if (paginatorElement && !compact && rootRef.current) {
+      if (paginatorElement && rootRef.current) {
```

Comment:
This was a decision we made in UITK, but I don't think we should do this in Salt. I think the composition with Input/FF should be done as a pattern and shown as an example. It would simplify a lot of our code